python_modules/dagit/dagit/webapp/src/SidebarSolidInfo.tsx

```diff
@@ -31,17 +31,24 @@ class PythonNotebookButton extends React.Component<{ path: string }> {
     open: false
   };
 
+  componentDidMount() {
+    document.addEventListener("show-python-notebook", this.onClick);
```

Comment:
(This is really just for demo purposes - this code, and hopefully this whole harcoded button) will go away soon)